client/src/components/PreviewImageField/PreviewImageField.js

```diff
@@ -302,7 +307,13 @@ class PreviewImageField extends Component {
           {message.value}
         </div>
       );
-    } else if (progress === 100) {
+    } else if (errors) {
+      messageBox = (
+        <div className={`preview-image-field__message preview-image-field__message--${errors.type}`}>
+          {errors.value}
+        </div>
+      );
+    } else if (progress === 100 && !errors) {
```

Comment:
```suggestion
    } else if (progress === 100) {
```
We don't need to check again if there are errors since this is an `else if` after we already checked that.